test/integration/targets/ansible-doc/runme.sh

```diff
@@ -175,3 +175,8 @@ ansible-doc --list --module-path ./modules > /dev/null
 
 # ensure we dedupe by fqcn and not base name
 #[ "$(ansible-doc -l -t filter --playbook-dir ./ |grep 'b64decode' | wc -l)" -eq "2"]
+
+# with playbook dir, legacy should override
+ansible-doc -t filter split --playbook-dir ./ |grep histerical
```

Comment:
Not for the PR, just making a note for later: listing legacy plugins that shadow builtins still doesn't work.

Reply:
yep, just figured out how to give correct docs, while listing should work when doing ansible.legacy filtering, but not in general unless we do direct masking/overlay of legacy over builtin (which im tempted to ignore or do in follow up)